packages/support/lib/fs.js

```diff
@@ -36,21 +37,48 @@ const findRootCached = _.memoize(pkgDir.sync);
 const fs = {
   /**
    * Resolves `true` if `path` is _readable_, which differs from Node.js' default behavior of "can we see it?"
-   * @param {import('fs').PathLike} path
+   *
+   * On Windows, ACLs are not supported, so we only check for file existence.
+   * @param {PathLike} path
    * @returns {Promise<boolean>}
    */
   async hasAccess(path) {
     try {
-      await fsPromises.access(path, constants.R_OK);
-    } catch (err) {
+      if (isWindows()) {
+        await fsPromises.stat(path);
```

Comment:
Do you have a link that confirms fs.access does not work on Windows? This method was working just fine before?

Reply:
https://nodejs.org/api/fs.html

> On Windows, access-control policies (ACLs) on a directory may limit access to
a file or directory. The `fs.access()` function, however, does not check the
ACL and therefore may report that a path is accessible even if the ACL restricts
the user from reading or writing to it.

Reply:
this just means it does not properly work with NTFS ACL, although it should be just ok to verify if a file exists. Lets keep this method as is for now to avoid possible regressions as it is called in multiple places across Appium